routes/user.route.js

```diff
@@ -1,14 +1,50 @@
 const express = require('express');
 const Router = express.Router();
-
+const User = require('../models/user.model');
```

Comment:
db services will only remain in `services/` directory. For every route, there is to be a handler -> from `controllers/` directory. If the route is just for rendering, render it in the controller, else if it uses any micro services, call the services inside its controller function.

Responses and rendering to be done only in controllers.